ui/desktop/electron-app/config/cli.js

```diff
@@ -95,20 +95,20 @@ const extract = async (artifactPath, destination) => {
 
 module.exports = {
   setup: async () => {
-    if (process.env.BYPASS_CLI_SETUP) {
+    if (process.env.CLI_SETUP) {
```

Comment:
Shouldn't this check for `if (process.env.CLI_SETUP === 'true')` since it would be a string? Otherwise any flag would pass this check

Reply:
You ar right, this wask a mistake on my end, thanks for the catch!